zerver/tests/test_logging_handlers.py

```diff
@@ -61,7 +61,9 @@ def get_admin_zulip_handler(self) -> AdminNotifyHandler:
             if isinstance(h, AdminNotifyHandler)
         ][0]
 
-    def test_basic(self) -> None:
+    @patch('zerver.logging_handlers.try_git_describe')
+    def test_basic(self, mock_function: Any) -> None:
```

Comment:
One small tweak I'll make as I merge this: we try to avoid introducing `Any` where we don't have to. Here, a more meaningful type is `mock.MagicMock`.